vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Install and use packages on Linux
+
+## Description
+
+This tutorial guides you through setting up a new C++ "Hello World" project using the `fmt` library. You'll manage dependencies, build the application, and eventually run your project, all using vcpkg. This tutorial assumes you have vcpkg setup and ready to use. If not, check out our initial setup [tutorial](setup-vcpkg.md).
+
+### Prerequisites
+
+For this tutorial, you'll need to have `git`, `bash`, `CMake`, and a C++ compiler installed.
+
+## Setup the project
+
+Create a directory for your new project next to your vcpkg clone and navigate into it:
+
+```bash
+mkdir helloworld && cd helloworld
+```
+
+Initiate a new project within this directory, creating the manifest (`vcpkg.json`) and default configuration files (`vcpkg-configuration.json`). From within the `helloworld` directory, run:
```

Comment:
I think it is also hard to see the boundaries between instruction steps, the long explanation from one step runs into the next instruction without a break of some sort. Just by skimming I could miss one step in the tutorial.

I'd recommend at least using heading style

--- 

## Setup the project 

### Create the project directory

Create a directory for your new project next to your vcpkg clone and navigate into it:

`mkdir helloworld & cd helloworld`

### Create the project files 

Initiate a new project within....

## Add the `fmt` dependencies

---

You could take it further using "Step #" in the headings too:


## Step 1: setup the project 
### Step 1.1: create the project directory
### Step 1.2: create the project files 
## Step 2: add the `fmt` dependencies

Reply:
Originally, I had it set up with steps, but @AugP suggested a more conversational style without the steps. I could add some headings to provide some breaks within the tutorial.

Reply:
Formatting with steps sounds good to me. I agree with Victor's feedback that it would make it easier to follow.

Reply:
@AugP, can you confirm the direction?